src/db/postgres/lrsql/postgres/sql/delete.sql

```diff
@@ -61,3 +61,39 @@ DELETE FROM credential_to_scope
 WHERE api_key = :api-key
 AND secret_key = :secret-key
 AND scope = :scope::scope_enum;
+
+----------------------begin components of delete-actor-----
+-- :name delete-actor-and-dependents!
+-- :command :execute
+-- :result :affected
+DELETE FROM statement_to_statement 
+WHERE ancestor_id IN (
+  SELECT statement_id FROM statement_to_actor
+  WHERE actor_ifi = :actor-ifi
+)
+OR descendant_id IN (
+  SELECT statement_id FROM statement_to_actor
+  WHERE actor_ifi = :actor-ifi
+);
+
+DELETE FROM statement_to_activity WHERE statement_id IN (
+  SELECT statement_id FROM statement_to_actor
+  WHERE actor_ifi = :actor-ifi
+);
+
+DELETE FROM attachment WHERE statement_id IN (
+  SELECT statement_id FROM statement_to_actor
+  WHERE actor_ifi = :actor-ifi
+);
+
+DELETE FROM xapi_statement WHERE statement_id IN (
+  SELECT statement_id FROM statement_to_actor
+  WHERE actor_ifi = :actor-ifi
+);
+
+DELETE FROM agent_profile_document WHERE agent_ifi = :actor-ifi;
+
+DELETE FROM state_document WHERE agent_ifi = :actor-ifi;
+
+DELETE FROM actor where actor_ifi = :actor-ifi;
```

Comment:
Apologies for not noticing the first time, but this `where` should also be capitalized by convention.

Reply:
capitalized